application/src/main/java/org/togetherjava/tjbot/config/RSSFeed.java

```diff
@@ -0,0 +1,19 @@
+package org.togetherjava.tjbot.config;
+
+import com.fasterxml.jackson.annotation.JsonProperty;
+
+import javax.annotation.Nullable;
+
+import java.util.Objects;
+
+public record RSSFeed(@JsonProperty(value = "url", required = true) String url,
```

Comment:
Needs doc

Reply:
A doc has been added! ✅

Reply:
Params are missing, if you add doc in intellij, it should add them automatically.